### ISSUE TITLE
Improve CSS when background colors present

### ISSUE THREAD

**Issue by mathetos** [WordImpress/Give#1258]
## Issue Overview

Our styles do a relatively good job on themes that have white backgrounds. But on dark backgrounds there's always CSS issues. We don't need to force Give to always work, but there's one item that if we resolved the forms would at least still appear functional out of the box even with dark backgrounds.

We set a background color on two elements. Specifically:

* span.give-final-total-amount ( https://github.com/WordImpress/Give/blob/master/assets/scss/frontend/forms.scss#L305 )
* .give_error, .give_success ( https://github.com/WordImpress/Give/blob/master/assets/scss/frontend/forms.scss#L340 )
 
But we don't specify a COLOR for those. So when the theme has white text, those Amount and Error Messages aren't visible at all.

Bottom line is if we are setting a background color, we should also set a COLOR as well.

## Possible Solution
`.give_error, .give_success {
  background: #F9F9F9;
  color: #000000;
}`

`span.give-final-total-amount {
      background: #FFF;
      color: #000000;
}`
